### ISSUE TITLE
url in spec_helper comment

### ISSUE THREAD

**Issue by maangie** [rspec/rspec-core#1968]
about https://github.com/rspec/rspec-core/blob/master/lib/rspec/core/project_initializer/spec/spec_helper.rb#L61

the url http://teaisaweso.me/blog/2013/05/27/rspecs-new-message-expectation-syntax/ timeout.

```bash
% wget -t 1 http://teaisaweso.me/blog/2013/05/27/rspecs-new-message-expectation-syntax/
--2015-05-25 16:16:39--  http://teaisaweso.me/blog/2013/05/27/rspecs-new-message-expectation-syntax/
Resolving teaisaweso.me... 54.243.85.192
Connecting to teaisaweso.me|54.243.85.192|:80... failed: Connection timed out.
Giving up.
```

the page may move to http://www.teaisaweso.me/blog/2013/05/27/rspecs-new-message-expectation-syntax/ .

i'm sorry my little english.

**Comment by JonRowe** [comment]
Fixed by #1969

**Comment by maangie** [comment]
thx.
